backend/src/main/java/ai/verta/modeldb/authservice/RoleServiceUtils.java

```diff
@@ -1070,23 +1070,53 @@ public WorkspaceDTO getWorkspaceDTOByWorkspaceName(
       String vertaId = authService.getVertaIdFromUserInfo(currentLoginUserInfo);
       workspaceDTO.setWorkspaceId(vertaId);
       workspaceDTO.setWorkspaceType(WorkspaceType.USER);
+      workspaceDTO.setWorkspaceName(authService.getUsernameFromUserInfo(currentLoginUserInfo));
     } else {
       try {
         workspaceDTO.setWorkspaceId(new CollaboratorOrg(getOrgByName(workspaceName)).getId());
         workspaceDTO.setWorkspaceType(WorkspaceType.ORGANIZATION);
+        workspaceDTO.setWorkspaceName(workspaceName);
       } catch (StatusRuntimeException e) {
-        workspaceDTO.setWorkspaceId(
+        CollaboratorUser collaboratorUser =
             new CollaboratorUser(
-                    authService,
-                    authService.getUserInfo(
-                        workspaceName, ModelDBConstants.UserIdentifier.USER_NAME))
-                .getId());
+                authService,
+                authService.getUserInfo(workspaceName, ModelDBConstants.UserIdentifier.USER_NAME));
+        workspaceDTO.setWorkspaceId(collaboratorUser.getId());
         workspaceDTO.setWorkspaceType(WorkspaceType.USER);
+        workspaceDTO.setWorkspaceName(workspaceName);
       }
     }
     return workspaceDTO;
   }
 
+  @Override
```

Comment:
```suggestion
  /**
  * Given the workspace id and type, returns WorkspaceDTO which has the id, name and type fo the workspace.
  **/
  @Override
```

Reply:
pull and mvn package for it to be formatted correctly

Reply:
Added it and formatted